src/app/tabs/app-web3-provider.js

```diff
@@ -0,0 +1,31 @@
+import { Plugin } from '@remixproject/engine'
+import * as packageJson from '../../../package.json'
+
+export const profile = {
+  name: 'appWeb3Provider',
```

Comment:
Name it `web3Provider` or `remixProvider` as it's going to be exposed to other plugins.
- Either we make it "generic" like that this work in any IDE. Then call it `web3Provider`. This is what I recommand.
- Or we force plugins developers to use different version depending on the IDE and we call it `remixProvider`.

Reply:
see previous commit about the global aspect of this provider.
as we might have several providers in the future running at the same time, how to reflect that this provider is the global one? perhaps using `default` is more relevant

Reply:
By calling it web3Provider you say that it's the default one. If another plugin wants to expose there own version they'll have to name it differently. 
It's like solidity. If another plugin want to exposed their own version of Solidity Compiler (for any reason), they can, but they cannot call it "solidity".
If we use `app****` to say it's the default one, then we should do that for all of them...

Reply:
ok so let's name it `provider`